src/scss/components/toast/_index.scss

```diff
@@ -1,9 +1,9 @@
-@import (reference) 'vars';
+@import 'vars';
```

Comment:
Please use double quotes:

```scss
@import "vars";
```

Reply:
Please check the changes in latest commit.